### DISCUSSION TITLE
对$-$添加背景色的高度特别小

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#146] [Q&A]
你好，
`\colorbox{lightgray}{$\mathrm{L_i}$}中的\colorbox{lightgray}{$-$}为`
这行代码运行后的结果为：
![image](https://user-images.githubusercontent.com/45897471/169698082-0f64c78e-8ff9-4c93-bd7f-d5a492117a1d.png)
很奇怪，`-`的背景色的高度不够，我在空白模板测试是正常的，我们的模板哪里限制了这个。
谢谢

**Comment by xkwxdyy** [comment]
1. 请提供 MWE
2. 这有什么问题吗，盒子的 minimum 效果。就像你总不会用一个别墅去装一只蚂蚁。

非要撑起来的话，可以考虑 `\rule` 或者 `\vphantom` 之类的把它高度撑起来
